src/apps/vpn/cmake/macos.cmake

```diff
@@ -4,6 +4,10 @@
 
 set_target_properties(mozillavpn PROPERTIES OUTPUT_NAME "Mozilla VPN")
 
+# Force llvm to add DWARF info on Release builds
```

Comment:
What I think we really want to be using to get debugging symbols is to set `-DCMAKE_BUILD_TYPE=RelWithDebInfo` when invoking CMake on the command line.